app/src/test/java/org/mozilla/fenix/components/metrics/DefaultMetricsStorageTest.kt

```diff
@@ -273,6 +212,74 @@ class DefaultMetricsStorageTest {
         assertFalse(captureSlot.isCaptured)
     }
 
+    @Test
+    fun `GIVEN during second day of install WHEN checking to track serp ad clicked THEN event will be sent`() = runTest(dispatcher) {
+        every { settings.secondDayAdClickGrowthSent } returns false
+        val yesterday = System.currentTimeMillis() - (dayMillis + 1000)
+        installTime = yesterday
+
+        val result = storage.shouldTrack(Event.GrowthData.SerpAdClicked)
+
+        assertTrue(result)
+    }
+
+    @Test
+    fun `GIVEN before second day on install WHEN checking to track serp ad clicked THEN event will not be sent`() = runTest(dispatcher) {
+        every { settings.secondDayAdClickGrowthSent } returns false
+
+        val result = storage.shouldTrack(Event.GrowthData.SerpAdClicked)
+
+        assertFalse(result)
+    }
+
+    @Test
+    fun `GIVEN after second day of install WHEN checking to track serp ad clicked THEN event will not be sent`() = runTest(dispatcher) {
+        every { settings.secondDayAdClickGrowthSent } returns false
+        val threeDaysAgo = System.currentTimeMillis() - (dayMillis * 3)
```

Comment:
would you also want the `+ 1000` buffer here?

Reply:
Sure, that seems like a safe bet to me